docs/ref/contrib/postgres/functions.txt

```diff
@@ -30,3 +30,32 @@ Usage example::
     >>> from django.contrib.postgres.functions import TransactionNow
     >>> Article.objects.filter(published__lte=TransactionNow())
     <QuerySet [<Article: How to Django>]>
+
+
+``TrigramDistance``
+===================
+
+.. class:: TrigramDistance(expression, expression, **extra)
+
+.. versionadded:: 1.10
+
+Accepts a field name or expression, and a string or expression. Returns the
+trigram distance between the two arguments. You may want to also see the
+:lookup:`trigram_distance` lookup.
+
+To use it, you need to activate the `pg_trgm extension on PostgreSQL`_. The
+:class:`~django.contrib.postgres.operations.TrigramExtension` migration
+operation is available if you want to perform this activation using
+migrations).
```

Comment:
Sure; I took the wording from the existing `unaccent` documentation: https://github.com/django/django/blob/ecb59cc6579402b68ddfd4499bf30edacf5963be/docs/ref/contrib/postgres/lookups.txt#L16-L18 :)